db/migrate/20210909094348_add_category_to_content_pages.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToContentPages < ActiveRecord::Migration[6.1]
+  def change
+    add_column :content_pages, :category, :string
+  end
```

Comment:
I'd call it categories, so that a content_page can belong to more than one at a time.

Reply:
Can you think of another category we'd use?

Reply:
I'm not sure I want to mix together the concepts without more discussion.  For the purpose of this change each page has one category (either an article or not).  I would consider groupings of pages to be a different.

Reply:
Thinking, the reason you need a category attribute, is because ContentPage inherits from Article, but you still need to filter out the ContentPages

This is a sign that something is wrong !  I don't think a ContentPage is an Article